mesh/reward.go

```diff
@@ -6,25 +6,29 @@ import (
 	"math/big"
 )
 
-type RewardConfig struct {
+type Config struct {
+	// reward config
 	SimpleTxCost   *big.Int
 	BaseReward     *big.Int
 	PenaltyPercent *big.Int
 	TxQuota        uint32
 	RewardMaturity LayerID
+	// global mesh config
+	LayersPerEpoch LayerID
```

Comment:
kind of weird that the type is `LayerID`

Reply:
removed